x-pack/plugins/enterprise_search/common/constants.ts

```diff
@@ -39,6 +39,23 @@ export const ENTERPRISE_SEARCH_CONTENT_PLUGIN = {
   SUPPORT_URL: 'https://discuss.elastic.co/c/enterprise-search/',
 };
 
+export const ANALYTICS_PLUGIN = {
+  ID: 'analytics',
```

Comment:
realised that this id is very important! I believe this id needs to be added here aswell https://github.com/elastic/kibana/blob/0be138c0a9eb8ff093f4a58df87558153b812ae5/src/plugins/kibana_usage_collection/server/collectors/application_usage/schema.ts#L137

Im thinking about renaming it to something more relatable to enterprise search. Something like `enterpriseSearchAnalytics`. 

WDYT?

Reply:
Makes sense to me!